tst/PortingAssistantExtensionIntegTests/FileUtils.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace PortingAssistantExtensionIntegTests
+{
+    //Copied from PA standalone tests
+    class FileUtils
+    {
+        static string[] allowList = { ".cs", ".json", ".config", ".nupkg", ".targets"};
```

Comment:
this will include `PortSolutionResult.json` which contains `solutionPath` and `projectPath` hash, I'm not 100% sure if the hash would be different if the path varies.

Reply:
Its handled in the isValidFile method.